lib/IDE/CodeCompletion.cpp

```diff
@@ -893,6 +898,17 @@ static void addAnyTypeKeyword(CodeCompletionResultSink &Sink, Type T) {
   Builder.addTypeAnnotation(T, PrintOptions());
 }
 
+static void addClosureSignatureKeywords(CodeCompletionResultSink &Sink,
```

Comment:
This function name sounds to me like it always add closure signature keywords, but it only adds them if `DC` is actually a closure. I think it would be clearer if either
- the function was named `addClosureSignatureKeywordsIfApplicable` or
- the function took a `ClosureExpr *` instead of a `DeclContext *` (I think I prefer this slightly, but I’m also fine with the first approach if you want to avoid the `if` check in the callee)

Reply:
Renamed to `addClosureSignatureKeywordsIfApplicable`